ipatests/test_integration/test_cockpit.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2016  FreeIPA Contributors see COPYING for license
```

Comment:
nit: date should be 2024.

Reply:
2024 seems to be enough for US, 2016-2024 may be preferred in other countries.